x-pack/plugins/cases/public/components/create/form.tsx

```diff
@@ -55,9 +52,6 @@ const MySpinner = styled(EuiLoadingSpinner)`
   left: 50%;
   z-index: 99;
 `;
-export type SupportedCreateCaseAttachment = CommentRequestAlertType | CommentRequestUserType;
```

Comment:
I moved out these types to the common types since they are not strictly only for create a case.